app/apk/src/main/java/com/topjohnwu/magisk/ui/home/HomeFragment.kt

```diff
@@ -49,14 +51,17 @@ class HomeFragment : BaseFragment<FragmentHomeMd2Binding>(), MenuProvider {
         savedInstanceState: Bundle?
     ): View {
         super.onCreateView(inflater, container, savedInstanceState)
-
         // If titles are squished, hide icons
         with(binding.homeMagiskWrapper) {
             checkTitle(homeMagiskTitle, homeMagiskIcon)
         }
         with(binding.homeManagerWrapper) {
             checkTitle(homeManagerTitle, homeManagerIcon)
         }
+        //if device is quest then disable install button for safety
```

Comment:
```suggestion
        // If device is META Quest then disable install button for safety
```

To follow the style of the comments in the file